### PR TITLE
Update to include USCUID-UL

### PR DESCRIPTION
**Pull request by Akury83**
Update to include brief outline of USCUID-UL chips
Removed hidden block reading and writing from UFUID chips, as these do not have hidden blocks.

### CODE REVIEW

File: doc/magic_cards_notes.md [hunk 1]
```diff
@@ -1142,8 +1143,6 @@ All commands are available before sealing. After the sealing acts as a Mifare Cl
 * Magic wakeup: `40(7)`, `43`
   * Backdoor read main block: `30xx+crc`
   * Backdoor write main block: `A0xx+crc`, `[16 bytes data]+crc`
-  * Read hidden block: `38xx+crc`
```
Comment (Akury83) [inline doc/magic_cards_notes.md:1145]: UFUID chips don't have hidden blocks

**Comment by iceman1001** [comment]
well,  nice with the update,  but I am not sure why you are removing information.   is `USCUID-UL`  now the identification of `direct flavor 2` ?

**Comment by Akury83** [comment]
Yeah the direct flavour 2 responds with an ATS that indicates it’s a USCUID-UL, hence removing it and merging it there.

File: doc/magic_cards_notes.md (hunk 1 above)
Comment (Akury83) [reply doc/magic_cards_notes.md:1145]: Suggested deleted lines 1145 and 1146 are in section `Mifare Classic > Mifare Classic USCUID > UFUID`, well above the proposed new USCUID-UL section which begins at line 1971.

Comment (iceman1001) [reply doc/magic_cards_notes.md:1145]: yes,  and are you saying that no `Mifare Classic > Mifare Classic USCUID`  has hidden blocks or?

Comment (Akury83) [reply doc/magic_cards_notes.md:1145]: No, only the “cut down” varieties do not have the hidden blocks such as `UFUID` and `ZUID`. Looking at it now, code lines `1235` and `1236` should also be deleted.

Comment (iceman1001) [reply doc/magic_cards_notes.md:1145]: So,   it would need to have partion for those who has hidden blocks.

Comment (Akury83) [reply doc/magic_cards_notes.md:1145]: It does, on lines `869-870` as general info, lines `1051-1052` for the `FUID` variant, lines `1313-1314` for the `GDM` variant, and lines `1382-1383` for the `GDCUID` variant. The UFUID and ZUID variants do not have hidden blocks, hence the request to remove those lines only.

**Pull request merged**